platform/windows_x64/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022 AS download
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV EF_INSTALL_TEMP_DIR="C:\\engflow_install_temp"
+ENV SCRIPT_DEBUG="on"
+
+COPY "normalize_os_windows.ps1" "C:/normalize_os_windows.ps1"
+RUN "C:/normalize_os_windows.ps1"
```

Comment:
If I remember correctly, I used forward slashes to avoid escaping. Backslash is the escape character for most (but not all?) instructions in `Dockerfile`. Maybe either would work here, but it's going to be less readable with escape characters.

Many tools accept either `/` or `\` but sometimes not both. It's kind of trial-and-error to make things work.